pkg/apis/v1beta1/nodepool.go

```diff
@@ -137,6 +143,18 @@ const (
 	ConsolidationPolicyWhenUnderutilized ConsolidationPolicy = "WhenUnderutilized"
 )
 
+// DisruptionReason defines valid reasons for disruption budgets.
+// +kubebuilder:validation:Enum={underutilized,expired,emptied,drifted}
+type DisruptionReason string
+
+const (
+	DisruptionReasonUnderutilized DisruptionReason = "underutilized"
+	DisruptionReasonExpired       DisruptionReason = "expired"
+	DisruptionReasonEmpty         DisruptionReason = "emptied"
```

Comment:
Sugg: empty. It may not have ever had pods.

Reply:
good callout. Changed this